payu/experiment.py

```diff
@@ -558,13 +576,37 @@ def run(self, *user_flags):
         if curdir:
             os.chdir(curdir)
 
-        self.runlog.create_manifest()
-        if self.runlog.enabled:
-            self.runlog.commit()
-
         f_out.close()
         f_err.close()
 
+        self.finish_time = datetime.datetime.now()
+
+        info = get_job_info()
+
+        if info is None:
+            # Not being run under PBS, reverse engineer environment
+            info = {
+              'PAYU_PATH': os.path.dirname(self.payu_path)
+            }
+
+        # Add extra information to save to jobinfo
+        info.update(
+          {
+            'PAYU_RUN_ID': self.run_id,
+            'PAYU_CURRENT_RUN': self.counter,
+            'PAYU_N_RUNS':  self.n_runs,
+            'PAYU_JOB_STATUS': rc,
+            'PAYU_START_TIME': self.start_time.isoformat(),
+            'PAYU_FINISH_TIME': self.finish_time.isoformat(),
+            'PAYU_WALLTIME': "{0} s".format(
+              (self.finish_time - self.start_time).total_seconds()
+             ),
+          }
+        )
```

Comment:
Can you 4-space this indentation?

Reply:
Heh. I only just changed it to that as I thought it's not code indentation, so might be good to differentiate, and PEP8 didn't complain so I figured it was ok.